heapster-saw/src/Verifier/SAW/Heapster/Parser.y

```diff
@@ -223,6 +225,11 @@ lifetime ::                                     { Maybe AstExpr         }
   :                                             { Nothing               }
   | '[' expr ']'                                { Just $2               }
 
+lifetimes ::                                    { [AstExpr]             }
+  :                                             { []                    }
+  | '[' ']'                                     { []                    }
+  | '[' list1R(expr) ']'                        { $2                    }
```

Comment:
Actually looking further, I think this can just be the following

```haskell
lifetimes ::                                    { [AstExpr]             }
  :                                             { []                    }
  | '[' list(expr) ']'                          { $2                    }
  ```

Reply:
Whoops, yes, I didn't understand that the `R` meant reversed. I thought it meant right-associated, and didn't look hard enough at it. Thanks for the catch!

Reply:
I will also make sure it is in sync with master.